frontend-next-migration/next.config.js

```diff
@@ -39,31 +39,31 @@
 //     withBundleAnalyzer,
 // ], nextConfig);
 
-
 // module.exports = nextConfig;
 const path = require('path');
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  images: {
+    remotePatterns: [
+      { hostname: process.env.NEXT_PUBLIC_API_DOMAIN },
+      { hostname: 'hips.hearstapps.com' },
+      { hostname: 'www.thesprucepets.com' },
+      { hostname: 'heroes.ts.unsplash.com' },
+      { hostname: 'raw.githubusercontent.com' },
+      { hostname: 'localhost' }, // Add localhost for development
+      { hostname: new URL(process.env.NEXT_PUBLIC_STRAPI_HOST).hostname },
```

Comment:
use envHelper

Reply:
is here some problem to with using the envHelper?

Reply:
in envHelper you have added : "strapiHost" , which is the "process.env.NEXT_PUBLIC_STRAPI_HOST", so just use envHelper.strapiHost instead ?